pyproject.toml

```diff
@@ -264,10 +249,16 @@ lint.select = [
     "NPY201", # numpy 2.0 compatibility (see https://numpy.org/devdocs/numpy_2_0_migration_guide.html#ruff-plugin) -- Explicitly listed to view preview rule errors
 ]
 lint.ignore = [
+    # formatting related ignores
+    # https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
+    "W191", # tab-identation
```

Comment:
I think we do want to enforce no tabs in python files.

Reply:
@billdirks 
Ruff recommends disabling these rules when using the formatter.

https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules

>Ruff's formatter is designed to be used alongside the linter. However, the linter includes some rules that, when enabled, can cause conflicts with the formatter, leading to unexpected behavior. When configured appropriately, the goal of Ruff's formatter-linter compatibility is such that running the formatter should never introduce new lint errors.

Reply:
Oh, I misread this as ruff formatter configuration but these are turning off linter rules. I'm good with this.